api/v1alpha2/istio_structs.go

```diff
@@ -15,6 +15,11 @@ type Config struct {
 
 	// Defines a list of external authorization providers.
 	Authorizers []*Authorizer `json:"authorizers,omitempty"`
+
+	// GatewayExternalTrafficPolicy defines the external traffic policy for Istio Ingress Gateway Service. Valid configurations are "Local" or "Cluster". An external traffic policy set to "Local  preserves the client IP in the request, but also introduces the risk of unbalanced traffic distribution.
```

Comment:
Looks like my suggestion had a bad spacing 🙄. Now we also need to regenerate the CRD again 🙈 
```suggestion
	// GatewayExternalTrafficPolicy defines the external traffic policy for Istio Ingress Gateway Service. Valid configurations are "Local" or "Cluster". An external traffic policy set to "Local preserves the client IP in the request, but also introduces the risk of unbalanced traffic distribution.
```